App.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import {Home, Login} from './src/containers';
+import { StyleSheet, TextInput} from 'react-native';
+import {Home, Login, SignUp} from './src/containers';
 
 const App = () => {
-  return <Login/>;
+  return <SignUp/>
```

Comment:
Codacy has a fix for the issue: [Replace `/>` with `·/>;`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
  return <SignUp />;
```